gamestonk_terminal/portfolio/README.md

```diff
@@ -0,0 +1,40 @@
+#PORTFOLIO
+
+This was developed to interface with Robinhood.  
+
+Once this screen is accessed, the first command to be run is 
+````
+login
+````
+Which will use your robinhood login information, which should be stored as environment variables or in the config_terminal.py
+
+Currently there are 2 options:
+* [hold](#hold)
+    * Look at current (stock only) holdings
+* [rhhist](#rhhist)
+    * Get and plot rhhistorical portfolio
+
+## hold <a name="hold"></a>
+
+Displays current holdings to the console:
+
+````
+usage: hold
+````
+There are no additional flags.  This just prints all current stonks, their last price, previous close, your equity and
+the % change from previous close.
+
+Example:
+![hold](https://user-images.githubusercontent.com/18151143/111685384-3c6ab080-87fe-11eb-80ce-9b256c396bf2.png)
+## rhhist <a name="rhhist"></a>
+Display your RH portfolio based on provided interval and span.  Based on the API data availablility, plotted as a candlestick chart.
+````
+usage: rhhist [-s --span] [-i --interval]
+````
+* -s/--span : How long to look back.  Defaults to the day. Options:
+    * [day, week, month, 3month, year, 5year, all]
+* -i/--interval: Data resolution. Defaults to 10minute intervals. Options:
+    * [5minute, 10minute, hour, day, week]
+    
+Example Default Output:
+![rhhist](https://user-images.githubusercontent.com/18151143/111685390-3d9bdd80-87fe-11eb-90f9-7ca8b0c1e7f8.png)
```

Comment:
Instead of a screenshot of the plot, I would click on the cd icon in order to save the image. Usually I make it a as wide as my screen and don't change the height, I've noticed that they end up looking nice with a good ratio.

Reply:
I will look at that.  Im noticing that the ppl title is kinda squished up top as well.

I was also playing with trying plotly for the figure (since graph objects has candle stick built in) - but using those figures opened a browser instead of an image - so I didn't go into it further.

Reply:
Yea, ideally for now I'd strongly prefer us to focus on matplotlib figures. In the future we may follow the DASH road, where plotly would defo come in place.